src/components/AddItems.js

```diff
@@ -65,66 +78,96 @@ const AddItems = () => {
   };
 
   return (
-    <div>
-      <h1>Add Your Items!</h1>
-      <form onSubmit={handleSubmit}>
-        <label className="label-1" htmlFor="item-name">
-          {' '}
-          Item Name:
-        </label>
-        <input
-          className={errorMessage ? 'error' : ''}
-          type="text"
-          placeholder="Add your item here"
-          name="itemName"
-          value={formData.itemName}
-          onChange={updateInput}
-        />
-        {errorMessage ? <p className="error">{errorMessage}</p> : null}
-        <br />
-        <br />
-        <fieldset className="fieldset">
-          <legend>Time Frame</legend>
-          <label htmlFor="lastEstimate">
-            {' '}
-            How soon will you buy this again?
-          </label>
-          <br />
-          <input
-            type="radio"
-            id="soon"
-            name="lastEstimate"
-            checked={formData.lastEstimate == 7}
-            value="7"
-            onChange={updateInput}
-          />
-          <label htmlFor="soon"> Soon</label>
-          <br />
-          <input
-            type="radio"
-            id="kinda-soon"
-            name="lastEstimate"
-            value="14"
-            checked={formData.lastEstimate == 14}
-            onChange={updateInput}
-          />
-          <label htmlFor="kinda-soon"> Kinda Soon</label>
-          <br />
-          <input
-            type="radio"
-            id="not-soon"
-            name="lastEstimate"
-            value="30"
-            checked={formData.lastEstimate == 30}
-            onChange={updateInput}
-          />
-          <label htmlFor="not-soon">Not Soon</label>
-          <br />
-        </fieldset>
-        <br />
-        <input type="submit" value="Add Item" className="input-1" />
-      </form>
-    </div>
+    <Box bg="brand.600">
+      <Center textStyle="roundedCorners">
+        <Text textStyle="h2" orientation="horizonal">
+          Add Your Items!
+        </Text>
+      </Center>
+      <Center bg="white">
+        <form onSubmit={handleSubmit}>
+          <Box my={10}>
+            <FormLabel textAlign="center" fontSize="20px" htmlFor="item-name">
```

Comment:
There is a missmatch between htmlFor (item-name) and the  name of the input (itemName) making an accessibility tester think that there is no label for the input.

Reply:
Thanks! We had to wrap the input in the form :)